cmd/snap/cmd_run_test.go

```diff
@@ -712,11 +745,11 @@ func (s *RunSuite) TestAntialiasHappy(c *check.C) {
 	app, outArgs = snaprun.Antialias("alias", inArgs)
 	c.Check(app, check.Equals, "an-app")
 	c.Check(outArgs, check.DeepEquals, []string{
-		"99", // COMP_TYPE (no change)
-		"99", // COMP_KEY (no change)
-		"11", // COMP_POINT (+1 because "an-app" is one longer than "alias")
-		"2",  // COMP_CWORD (no change)
-		" ",  // COMP_WORDBREAKS (no change)
+		"99",                    // COMP_TYPE (no change)
```

Comment:
is this a gofmt change? it seems unrelated

Reply:
Quite possible, 1.11 changed the maximum blank fill length between a value and a comment. I'm quite sure this will break our gofmt checks on travis too.

Reply:
That's the change: https://golang.org/doc/go1.11#gofmt

Reply:
Yep, go fmt, I was actually running it with go 1.12.x and indeed it made travis fail with 1.10. Going to reformat with 1.10.